src/main/java/com/flowable/flowableboot/FlowablebootApplication.java

```diff
@@ -17,7 +17,7 @@ public static void main(String[] args) {
 
 	@Bean
 	public CommandLineRunner init(final FlowableService flowableService) {
-		return strings -> flowableService.createDemoUsers();
```

Comment:
if this is returning null now, do we still need this Bean for anything? I removed it and rebuilt ...everything seemed to run fine without it. Let me know if I missed something.

If you remove it, I think there are some imports that could be cleaned up